.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+# https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels: [ ]     # Do not add default "dependency" label
+
+  - package-ecosystem: "maven"
+    directory: "/"
```

Because it is not listed as a submodule in the root POM (and there are no dependencies to miss there anyway)
```suggestion
    directory: "/its"
```